reproschema/redcap2reproschema.py

```diff
@@ -184,7 +185,7 @@ def parse_field_type_and_value(field):
         elif value_type == "xsd:date" and field_type == "text":
             input_type = "date"
     elif field_type == "yesno":
-        value_type = "xsd:boolen"
+        value_type = "xsd:boolean"
```

Comment:
Am I correct this is a typo?
@yibeichan

Reply:
yes, this should be boolean